templates/firefox-ios.xliff

```diff
@@ -3250,6 +3250,49 @@ Private Tab</source>
       </trans-unit>
     </body>
   </file>
+  <file original="Shared/Supporting Files/en.lproj/Bookmarks.strings" datatype="plaintext" source-language="en" target-language="en">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="15.0.1" build-num="15A507"/>
+    </header>
+    <body>
+      <trans-unit id="Bookmarks.Menu.AllBookmarks.v131" xml:space="preserve">
+        <source>All</source>
+        <note>When navigating through the bookmarks menu and bookmark folders, a back button with an `All` (bookmarks) label is present to take the user to the top level bookmarks menu.</note>
+      </trans-unit>
+      <trans-unit id="Bookmarks.Menu.BookmarksTopLevelTitle.v131" xml:space="preserve">
+        <source>Bookmarks</source>
+        <note>Name of the top level bookmarks folder present in the folder selection menu of the`Edit Bookmark` screen</note>
+      </trans-unit>
+      <trans-unit id="Bookmarks.Menu.DeleteFolder.v131" xml:space="preserve">
+        <source>Delete Folder</source>
+        <note>When a folder is longpressed in the bookmarks menu, a `Delete Folder` button is present.</note>
+      </trans-unit>
+      <trans-unit id="Bookmarks.Menu.DeletedBookmark.v131" xml:space="preserve">
+        <source>Deleted "%@"</source>
```

Comment:
Couldn't track down how landed this string for some reason.
@ih-codes Could you help find someone to fix this please?
We would need [typographic](https://acorn.firefox.com/latest/content/punctuation-7J3JVM4o#section-apostrophes-and-quotation-marks-65) quotes in this instance. Thanks!

Reply:
String are mostly added in one file: https://github.com/mozilla-mobile/firefox-ios/blob/main/firefox-ios/Client/Frontend/Strings.swift

String in question: https://github.com/mozilla-mobile/firefox-ios/blob/main/firefox-ios/Client/Frontend/Strings.swift#L166

PR: github.com/mozilla-mobile/firefox-ios/blob/main/firefox-ios/Client/Frontend/Strings.swift#L166

@DanielDervishi

Reply:
Thanks @flodolo, @DanielDervishi just [merged in a fix](https://github.com/mozilla-mobile/firefox-ios/pull/21666) for this.

@Delphine You should be good to pull in the strings again!